settings.gradle.kts

```diff
@@ -121,15 +122,12 @@ dependencyResolutionManagement {
     // distribution. These libs can be depended on during compilation, or bundled as part of
     // runtime.
     create("libs") {
-      // The HAPI API version to use, this need to match the tag set on gitRepositories above
```

Comment:
Not sure best way here, this used to be used for the tag/branch in gitRepositories block. Not sure best method, want to make it easy for people to manage and find.

Reply:
I Put this back, but I am not sure if this is how you want it. There is now one `hapiVersion` that is used as the branch/tag name here and is written as `hapi.proto.version` version in the metadata.